apps/dashboard/config/configuration_singleton.rb

```diff
@@ -74,8 +74,9 @@ def string_configs
       :rclone_extra_config            => nil,
       :default_profile                => nil,
       :project_size_timeout           => '15',
-      :novnc_default_compression   => '6',
-      :novnc_default_quality       => '2'
+      :novnc_default_compression      => '6',
+      :novnc_default_quality          => '2',
+      :plugins_directory              => '/var/www/ood/apps/plugins'
```

Comment:
This should probably be `/etc/ood/config/plugins`. Stuff in `/var/www/ood` will be overwritten on updates.

Reply:
fixed